### PR TITLE
V1 release

### PR DESCRIPTION
**Pull request by talha0324**
Feature list:
- Patch resources with the provided config
- Check if pods can be evicted and rescheduled onto other nodes

### CODE REVIEW

**Comment by stakater-user** [comment]
@talha0324 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/k8s-cost-optimizer/commit/2c9b939038454c22298ff871094457aaabd490b1/checks) has Failed!

**Comment by stakater-user** [comment]
@talha0324 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/k8s-cost-optimizer/commit/7c96925af214da6fd315a56151df38b6a7bfc0a7/checks) has Failed!

**Comment by stakater-user** [comment]
@talha0324 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/k8s-cost-optimizer/commit/fd81a4185ce2f9773d8dbc53c5a83dbb9d4a39ee/checks) has Failed!

**Comment by stakater-user** [comment]
@talha0324 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/k8s-cost-optimizer/commit/1520a185d2172957e1cf0b00ffdb43f61eb0c6b8/checks) has Failed!

**Comment by stakater-user** [comment]
@talha0324 Image is available for testing. `docker pull stakater/k8s-cost-optimizer:SNAPSHOT-PR-1-82a7ba5f`

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    GOOS=${TARGETOS} \
+    GOARCH=${TARGETARCH} \
+    GOPROXY=${GOPROXY} \
+    GOPRIVATE=${GOPRIVATE} \
+    GO111MODULE=on \
+    go build -mod=mod -a -o k8s-cost-optimizer main.go
+
+# RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build
+
+FROM scratch
```
Comment (tanalam2411) [inline Dockerfile:27]: Should we use `scratch` as the base image?

**Review by tanalam2411 — CHANGES_REQUESTED** [review]

**Comment by stakater-user** [comment]
@talha0324 Image is available for testing. `docker pull stakater/k8s-cost-optimizer:SNAPSHOT-PR-1-9e4ed385`

File: Dockerfile (hunk 1 above)
Comment (tanalam2411) [reply Dockerfile:27]: I think scratch is good for now

**Review by tanalam2411 — APPROVED** [review]

**Pull request merged**